docs/docs/tutorials/using_dask.md

```diff
@@ -31,21 +32,24 @@ Dask consists of 3 main components `client`, `scheduler`, and `workers`.
 
 The `client` interacts with both `scheduler` (sends instructions) and `workers` (collects results).
 
-Check out the [Dask Gateway documentation][dask-gateway] for a full explanation.
+Check out the [Dask documentation][dask-docs] and the [Dask Gateway documentation][dask-gateway] for a full explanation.
 
 </details>
 
-## Step 1 - Setting up Dask Gateway
+## Step 1 - Set up Dask Gateway
 
-Let's start by creating a Jupyter notebook. Select an environment from the `Select kernel` dropdown menu
+Let's start with a fresh Jupyter notebook.
+Select an environment from the `Select kernel` dropdown menu
 (located on the top right of your notebook).
+On a default Nebari deployment, you can select the `filesystem/dask` environment.
```

Comment:
Need to double check the default dask env name.

Reply:
Thanks, updated!